mypy_django_plugin/lib/helpers.py

```diff
@@ -474,4 +474,9 @@ def resolve_lazy_reference(
 
 
 def is_model_type(info: TypeInfo) -> bool:
-    return info.metaclass_type is not None and info.metaclass_type.type.fullname == fullnames.MODEL_METACLASS_FULLNAME
+    return info.metaclass_type is not None and (
+        # Using the model metaclass shipped by Django
+        info.metaclass_type.type.fullname == fullnames.MODEL_METACLASS_FULLNAME
+        # Using a custom model metaclass that inherits above
+        or info.metaclass_type.type.has_base(fullnames.MODEL_METACLASS_FULLNAME)
```

Comment:
I think that you can remove `info.metaclass_type.type.fullname == fullnames.MODEL_METACLASS_FULLNAME` part, since `has_base` goes for all `mro` entries, including itself.